pkg/actions/cluster/owned.go

```diff
@@ -100,6 +100,14 @@ func (c *OwnedCluster) Delete(_ time.Duration, wait, force bool) error {
 			}
 			oidcSupported = false
 		}
+
+		allStacks, err := c.stackManager.ListNodeGroupStacks()
+		if err != nil {
+			return err
+		}
+		if err := drainAllNodegroups(c.cfg, c.ctl, c.stackManager, clientSet, allStacks); err != nil {
+			return err
+		}
```

Comment:
since this is identical in `owned` and `unowned` could it be moved into the `sharedResourced` function, or some new function in `delete.go`?

Reply:
actually I see you re-use the stacks down the line... nevermind